### ISSUE TITLE
Add Holden application to metadata generator

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#6111]
# Story

As a UIHC MarCom developer, I would like to integrate SSO on the cancer.uiowa.edu website so that users authenticating to the website can log in with their HawkID similar to other campus D9 Drupal websites.

# Additional Information

This is for the holden application.

We need to add the application to the metadata generator.
https://git.uiowa.edu/its-web/data.its.uiowa.edu/-/blob/main/app/Http/Controllers/ShibbolethController.php

# Proposed solution

- [x] Add application to metadata generator
- [x] Deploy
- [x] Communicate with Sean Davis that this is done

**Comment by joewhitsitt** [comment]
https://data-test.its.uiowa.edu/shibboleth/drupal (crow)
https://data.its.uiowa.edu/shibboleth/drupal (vulture)

**Comment by joewhitsitt** [comment]
Brian is sending me the details for this

**Comment by joewhitsitt** [comment]
Waiting on response from Sean Davis to confirm we set it up correctly.

**Comment by briand44** [comment]
>Waiting on response from Sean Davis to confirm we set it up correctly.

I would just call it done. I've confirmed the domains exist in the metadata. We can open a new issue if something comes up.
